dronecore/plugins/telemetry.py

```diff
@@ -14,40 +152,99 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
         # Create observables
+        self._create_in_air_observable()
         self._create_armed_observable()
+        self._create_position_observable()
+        self._create_health_observable()
+        self._create_battery_observable()
 
     def _setup_stub(self, channel):
         """
         Setup the telemetry stub
         """
         self._stub = telemetry_pb2_grpc.TelemetryServiceStub(channel)
 
+    def _create_in_air_observable(self):
+        """
+        Creates inair observable
+        """
+        stream = self._stub.SubscribeInAir(
+            telemetry_pb2.SubscribeInAirRequest())
+
+        self._in_air_observable = self._create_stream_observable(
+                response.is_in_air for response in stream)
+
     def _create_armed_observable(self):
         """
         Creats the inair observable
         """
-        def f(observable):
-            stream = self._stub.SubscribeArmed(
-                telemetry_pb2.SubscribeArmedRequest()
-            )
+        stream = self._stub.SubscribeArmed(
+            telemetry_pb2.SubscribeArmedRequest())
 
-            try:
-                while True:
-                    # @TODO
-                    # observable.on_next(
-                    #     stream.receive().is_in_air
-                    # )
-                    pass
-            except Exception as e:
-                observable.on_error(e)
+        self._armed_observable = self._create_stream_observable(
+                response.is_armed for response in stream)
 
-            observable.on_completed()
+    def _create_position_observable(self):
+        """
+        Create the position ovservable
```

Comment:
ovservable -> observable :). In all the other functions, too ;-)

Reply:
damnit! Thanks!